cristin-import/src/test/java/cucumber/JournalFeatures.java

```diff
@@ -182,4 +182,12 @@ public void theNvaResourceHasAReferenceObjectWithDoiEqualTo(String expectedDoi)
         URI actualDoi = reference.getDoi();
         assertThat(actualDoi, is(equalTo(URI.create(expectedDoi))));
     }
+
+    @Given("that the Journal Article entry has an empty \"publisherName\" field")
+    public void thatTheJournalArticleEntryHasAnEmptyPublisherNameField() {
+        scenarioContext.getCristinEntry()
+                        .getJournalPublication()
+                        .getJournal()
+                        .setPublisherName(null);
```

Comment:
Make this a constant like EMPTY_PUBLISHER_NAME to keep it like dummy test values elsewhere